x-pack/plugins/observability_solution/apm/public/utils/get_signal_type.ts

```diff
@@ -16,3 +16,9 @@ export function isApmSignal(dataStreamTypes: EntityDataStreamType[]) {
 export function isLogsSignal(dataStreamTypes: EntityDataStreamType[]) {
   return dataStreamTypes.includes(EntityDataStreamType.LOGS);
 }
+
+export function isLogsOnlySignal(signalTypes: EntityDataStreamType[]) {
+  return false;
```

Comment:
ah It took me a few minutes to figure out why the empty messages don't render on the screen, It would be better to keep this PR separate from the work related to adding telemetry.